akka-cluster-typed/src/main/protobuf/ClusterMessages.proto

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (C) 2014-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package akka.cluster.typed;
+
+option java_package = "akka.cluster.typed.internal.protobuf";
+option optimize_for = SPEED;
+
+
+message ReceptionistEntry {
+  required string actorRef = 1;
+  required uint64 systemUid = 2;
```

Comment:
Does making a protobuf field 'required' mean we can never make it optional in the future, or does that remain possible (though perhaps inconvenient)?

Reply:
Might work to change it to optional later but can’t be used as optional in first rollout, must be a two step rollout I think.